activity_browser/ui/tables/models/base.py

```diff
@@ -420,18 +420,24 @@ class ABSortProxyModel(QSortFilterProxyModel):
 
     See this for context: https://github.com/LCA-ActivityBrowser/activity-browser/pull/1151
     """
-    def lessThan(self, left, right) -> bool:
+    def lessThan(self, left: QModelIndex, right: QModelIndex) -> bool:
         """Override to sort actual data, expects `left` and `right` are comparable.
 
         If `left` and `right` are not the same type, we check if numerical and empty string are compared, if that is the
         case, we assume empty string == 0.
         Added this case for: https://github.com/LCA-ActivityBrowser/activity-browser/issues/1215"""
-        left_data = self.sourceModel().data(left, 'sorting')
-        right_data = self.sourceModel().data(right, 'sorting')
+        left_data = self.sourceModel().data(left)
+        right_data = self.sourceModel().data(right)
+        if not left_data and not right_data:
```

Comment:
Please make all follow-up checks `elif`, that makes it clearer this is one test case, not multiple in a row.